terminus/geometry/bounding_box.py

```diff
@@ -0,0 +1,33 @@
+from geometry.point import Point
+
+
+class BoundingBox(object):
+
+    def _normalize(self):
+        new_origin = Point(min(self.origin.x, self.corner.x),
```

Comment:
Since ordering points in 3D (and 2D for that matter) is tricky, I would say we define the min of two points as the min of each component (same for max)